### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by phelipe** [JuliaComputing/JuliaDB.jl#223]
JuliaDB throws error in installation
```
(v1.0) pkg> add JuliaDB
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package JuliaDB [a93385a2]:
 JuliaDB [a93385a2] log:
 ├─possible versions are: [0.1.0-0.1.5, 0.2.0-0.2.3, 0.3.0-0.3.7, 0.4.0-0.4.4, 0.5.0, 0.6.0, 0.7.0-0.7.2, 0.8.0-0.8.5] or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions [0.1.0-0.1.5, 0.2.0-0.2.3, 0.3.0-0.3.7, 0.4.0-0.4.4, 0.5.0, 0.6.0, 0.7.0-0.7.2, 0.8.0-0.8.5]
 └─restricted by julia compatibility requirements to versions: uninstalled — no versions left


```

**Comment by joshday** [comment]
0.7 and 1.0 support is a work in progress.  Almost all the JuliaDB dependencies are ready to go, so the wait shouldn't be much longer.

**Comment by pasta41** [comment]
Is there an update on this? Thanks.

**Comment by joshday** [comment]
This is where the work is going on:  https://github.com/JuliaComputing/JuliaDB.jl/pull/224

**Comment by EdoAlvarezR** [comment]
I'm getting the same error with [Julia 0.7.0 binary](https://julialang.org/downloads/oldreleases.html) out of the box.

```julia
(v0.7) pkg> add JuliaDB
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package JuliaDB [a93385a2]:
 JuliaDB [a93385a2] log:
 ├─possible versions are: [0.1.0-0.1.5, 0.2.0-0.2.3, 0.3.0-0.3.7, 0.4.0-0.4.4, 0.5.0, 0.6.0, 0.7.0-0.7.2, 0.8.0-0.8.5, 0.9.0, 0.10.0, 0.11.0-0.11.2] or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions [0.1.0-0.1.5, 0.2.0-0.2.3, 0.3.0-0.3.7, 0.4.0-0.4.4, 0.5.0, 0.6.0, 0.7.0-0.7.2, 0.8.0-0.8.5, 0.9.0, 0.10.0, 0.11.0-0.11.2]
 └─restricted by julia compatibility requirements to versions: uninstalled — no versions left
```

(I'm trying to get dig through some old databases I had developed before Julia 1.0 was released, that's why I'm trying to get 0.7 to work)

**Comment by EdoAlvarezR** [comment]
Fixed, I just needed to specify the commit I wanted to build manually to by pass the error:

```julia
(v0.7) pkg> add JuliaDB #32ac314
```
